src/tools/inspect/inspect.hs

```diff
@@ -46,12 +46,7 @@ usage = do
 
 main :: IO ()
 main = commandLineArgs <$> getArgs >>= \case
-    Just (db, repo, args) -> do
-        when (null args) usage
+    Just (db, repo, subcmd:_:_:args) ->
```

Comment:
I think this one should be `Just (db, repo, subcmd:args) ->` instead.  Any other thoughts?